components/brave_rewards/resources/ui/storage.ts

```diff
@@ -56,7 +56,9 @@ export const defaultState: Rewards.State = {
   },
   pendingContributionTotal: 0,
   grants: [],
-  currentGrant: undefined
+  currentGrant: undefined,
+  adsNotificationsReceived: 0,
```

Comment:
we should move this into `adsData` object, so that we have all ads data in one place

Reply:
@ryanml ^

Reply:
👍